image/run-scripts.sh

```diff
@@ -29,8 +29,10 @@ echo ''
 
 
 echo 'Running ktlint...'
+# Strip out any --reporter arguments supplied by the user
+INPUT_ARGUMENTS = $INPUT_ARGUMENTS#--reporter*[[:space:]]
```

Comment:
I tried this in my bash with a simple example and it seems not to work:
```bash
> INPUT_ARGUMENTS="--android --debug --reporter=json,output=/a/b/c --color"
> echo $INPUT_ARGUMENTS
--android --debug --reporter=json,output=/a/b/c --color
> INPUT_ARGUMENTS=$INPUT_ARGUMENTS#--reporter*[[:space:]]
> echo $INPUT_ARGUMENTS
--android --debug --reporter=json,output=/a/b/c --color#--reporter*[[:space:]]
```
I suggest we use the [substitution](https://devhints.io/bash#parameter-expansions) command:
```bash
> INPUT_ARGUMENTS="--android --debug --reporter=json,output=/a/b/c --color"
> echo $INPUT_ARGUMENTS
--android --debug --reporter=json,output=/a/b/c --color
> INPUT_ARGUMENTS=${INPUT_ARGUMENTS/--reporter*[[:space:]]/}
> echo $INPUT_ARGUMENTS
--android --debug --color
```

__Note__ that when assigning values to a variable there should be no spaces surrounding the `=`.

Reply:
Interesting that it didn't work for you! I passed in my own reporter arguments via Actions, and it seemed to remove it just fine. I'll replace it with what you suggested when I get the chance, thanks for catching that!